Ginger/Ginger/BusinessFlowWindows/ucBusinessFlowMap.xaml.cs

```diff
@@ -0,0 +1,130 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using System.Windows;
+using System.Windows.Controls;
+using GingerWPF.UserControlsLib.UCTreeView;
+using Amdocs.Ginger.Repository;
+using amdocs.ginger.GingerCoreNET;
+using GingerCore;
+using Amdocs.Ginger.Common.Enums;
+using System.Collections.Generic;
+using Ginger.SolutionWindows.TreeViewItems;
+using System;
+using Amdocs.Ginger.Common;
+
+namespace Ginger.BusinessFlowWindows
+{
+    /// <summary>
+    /// Interaction logic for ucBusinessFlowMap.xaml
+    /// </summary>
+    public partial class ucBusinessFlowMap : UserControl
+    {
+        Object mObjectElementType;
+        string mElementTypeFieldName;
+
+        BusinessFlow mBusinessFlow;
+        public BusinessFlow BusinessFlow
+        {
+            get
+            {
+                return mBusinessFlow;
+            }
+            set
+            {
+                mBusinessFlow = value;
+                xBFTextBox.Text = mBusinessFlow.FilePath.Substring(0, mBusinessFlow.FilePath.LastIndexOf("\\")).Substring(mBusinessFlow.ContainingFolderFullPath.Length) + @"\" + mBusinessFlow.ItemName;
+                xGoToAutomateBtn.Visibility = Visibility.Visible;
+                mBusinessFlowRepositoryKey = new RepositoryItemKey();
+                mBusinessFlowRepositoryKey.Guid = mBusinessFlow.Guid;
+                mBusinessFlowRepositoryKey.ItemName = mBusinessFlow.Name;
+                if (mObjectElementType != null)
+                {
```

Comment:
you implemented SetValue for BFKey but not GetValue- meaning if you will map a BF to POM, Save and then reload the solution, go to same POM and click on "GoToAutomate" tab of mapped BF- will it shift to Automate tab? i belive not because mBusinessFlow is still null.
if i didn't missed anything and my scenario do exist so you need to implement the set of mBusinessFlow from Object and field you got on control constructor.

Reply:
No, i tested the scenario it is working as expected, i can show you the scenario

Reply:
Implemented the changes